boards/shared/Src/App/App_SharedSignal.c

```diff
@@ -0,0 +1,75 @@
+#include <assert.h>
+#include <stdlib.h>
+#include "App_SharedSignal.h"
+
+struct Signal
+{
+    // The last time this signal was low, in milliseconds
+    uint32_t last_time_low_ms;
+
+    // The last time this signal was high, in milliseconds
+    uint32_t last_time_high_ms;
```

Comment:
similar comment here about noting `was observed to be` instead of just `was`

Reply:
Fixed